src/pages/open-graph/[...docs].ts

```diff
@@ -0,0 +1,85 @@
+import type { CollectionEntry } from 'astro:content';
+import { OGImageRoute } from 'astro-og-canvas';
+import { allPages } from 'src/utils/content';
+
+// setup rtlLanguages here
+const rtlLanguages = new Set(['']);
+const getLangFromSlug = (slug: CollectionEntry<'docs'>['slug']) => slug.split('/')[0];
+
+// TODO: Setup the process.env to skip on non production builds
+/** Paths for all of our Markdown content we want to generate OG images for. */
+const paths = process.env.SKIP_OG ? [] : allPages;
```

Comment:
We need to setup this env var. Btw, this snipped I just yanked from upstream, so I'm not sure when it was used. But we can play around with something that fits our needs

Reply:
Looking at https://docs.netlify.com/configure-builds/environment-variables/#build-metadata maybe `CONTEXT` is what you can use to check this?

Reply:
That works. I looked over astro docs and, apparently, they only use SKIP_OG on a single lint script. I wonder why